lib/bolt/node/errors.rb

```diff
@@ -18,5 +18,22 @@ def kind
         'puppetlabs.tasks/connect-error'
       end
     end
+
+    class FileError < BaseError
+      def kind
+        'puppetlabs.tasks/task_file_error'
+      end
+    end
+
+    class EnvironmentVarError < BaseError
+      def initialize(var, val)
+        message = "Could not set environment variable '#{var}' to '#{val}'"
+        super(message, 'ENVVAR_ERROR')
```

Comment:
How is this issue code helpful vs `puppetlabs.tasks/environment-var-error`?

Reply:
I guess every BaseError has an issue code, so that makes sense.

Reply:
issue code is to help with reverse translation in the future. It should match the message template.